### PR TITLE
chore: remove dashboard onboarding message

### PR DESCRIPTION
**Pull request by Ziinc**
Removes dashboard onboarding message for more space on home page.
Part of reorganizing and improving home layout.

Screenshot 2023-09-21 at 11 22 22 PM

### CODE REVIEW

**Comment by chasers** [comment]
I was actually wanting to remove this too.  But I was thinking we put links to our libs and extensions there instead. And maybe a link to the latest Github Release?

**Comment by chasers** [comment]
And probably a link to the API (SwaggerUI) docs will be helpful here too.

**Comment by Ziinc** [comment]
Screenshot 2023-09-29 at 8 18 53 PM
Links have been added in. Did not list all as clients are being worked on.

File: lib/logflare_web/templates/source/dashboard.html.eex [hunk 1]
```diff
@@ -114,22 +114,57 @@
...
+
+        <li class="list-group-item tw-text-sm">
+          <%= link "Vercel", to: "https://docs.logflare.app/integrations/vercel/" %>
+        </li>
+        <li class="list-group-item tw-text-sm">
+          <%= link "Fly", to: "https://github.com/Logflare/fly-log-shipper" %>
+        </li>
+        <li class="list-group-item tw-text-sm">
+          <%= link "Postgres  FDW", to: "https://docs.logflare.app/integrations/postgres-fdw" %>
+          <span class="tw-text-xs tw-block">Javascript</span>
```
Comment (chasers) [inline lib/logflare_web/templates/source/dashboard.html.eex:133]:
```suggestion
          <span class="tw-text-xs tw-block">SQL</span>
```

**Pull request merged**